action.yml

```
@@ -14,6 +14,9 @@ inputs:
     description: "Number of seconds after which the action runs again if the mergable state is unknown."
   retryMax:
     description: "Number of times the action retries calculating the mergable state"
+ouputs:
+  prDirtyStatuses:
+    description: "Array of booleans which indicate whether a PR is dirty or not and where the index is the PR ID."
```

```suggestion
    description: "Object-map. The keys are pull request numbers and their values whether a PR is dirty or not."
```

Does this make more sense to you?